src/webui/www/private/scripts/dynamicTable.js

```diff
@@ -1225,6 +1225,18 @@ window.qBittorrent.DynamicTable = (function() {
                     if (~state.indexOf('paused'))
                         return false;
                     break;
+                case 'stalled':
+                    if(state != 'stalledUP' && state != 'stalledDL')
+                        return false;
+                    break;
+                case 'stalled_up':
+                    if(state != 'stalledUP')
```

Comment:
Please use `===`

Reply:
Can you please clarify? `===` is not used in any other case, why should it be used here?

Reply:
Maybe he meant `!==`?

Reply:
@Chocobo1 Still, other places are using `!=`. If if should be changed, I think it is better to do so in bulk in a new PR, much the same way the removal of the usage of `~` around here is being done separately.

Reply:
@Chocobo1 @Piccirello so, what will it be?

Reply:
@Chocobo1 @Piccirello ping